toolset/run-tests.py

```diff
@@ -162,6 +164,10 @@ def main(argv=None):
     parser.set_defaults(**defaults) # Must do this after add, or each option's default will override the configuration file default
     args = parser.parse_args(remaining_argv)
 
+    if args.init:
+        Scaffolding()
```

Comment:
Also, not sure this is the proper syntax, though it works. I feel like I created a class, but didn't assign it... not sure. Python+Classes confuses me.